src/AppInstallerCLICore/Workflows/WorkflowBase.h

```diff
@@ -29,6 +29,19 @@ namespace AppInstaller::CLI::Workflow
         PostExecution = 5000,
     };
 
+    enum class SearchPurpose

```

Comment:
nit: Now with the latest change that contains almost every command, I think SearchPurpose could be called OperationType or a more generic name